test/bazel-karma-local-config.js

```diff
@@ -1,14 +1,14 @@
 /**
- * Karma configuration that is used by Bazel karma_web_test targets which do not
+ * concatjs configuration that is used by Bazel concatjs_web_test targets which do not
```

Comment:
Were the changes in this file intentional? They look like they might be an overzealous find & replace.